src/components/ArticleCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+import CardHeader from '@material-ui/core/CardHeader'
+import CardActionArea from '@material-ui/core/CardActionArea'
+import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import CardMedia from '@material-ui/core/CardMedia'
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography'
+import Divider from '@material-ui/core/Divider'
+import { styles } from '../theme/styles'
+import Link from '@material-ui/core/Link'
+
+const useStyles = makeStyles(styles)
+
+const ArticleCard = ({ article, spacing }) => {
+  const classes = useStyles()
```

Comment:
I know it's convention, but is there a benefit to declaring `useStyles` separately, vs the following?
```suggestion
const ArticleCard = ({ article, spacing }) => {
  const classes = makeStyles(styles);
```

Reply:
I only write it that way out of convention, though it seems either works. I recall having to use `makeStyles(styles)()` syntax in TypeScript for some reason having to do with the way types are evaluated.